src/CssInliner.php

```diff
@@ -419,9 +419,9 @@ private function normalizeStyleAttributes(\DOMElement $node): void
     {
         $normalizedOriginalStyle = \preg_replace_callback(
             '/-?+[_a-zA-Z][\\w\\-]*+(?=:)/S',
-            /** @param array<int, string> $m */
-            static function (array $m): string {
-                return \strtolower($m[0]);
+            /** @param array<int, string> $mixedCaseString */
+            static function (array $mixedCaseString): string {
+                return \strtolower($mixedCaseString[0]);
```

Comment:
Not sure about the new variable name.  I think originally `$m` stood for `$matches`.  It's not actually a string but a 1-length array of the regular expression matches (and I thought we were trying to avoid types within variable names). 

I would suggest something like `$propertyNameMatches` (because a property name is what is being matched here).

Reply:
Done.